polkadot/xcm/xcm-executor/src/config.rs

```diff
@@ -114,4 +110,8 @@ pub trait Config {
 
 	/// Transactional processor for XCM instructions.
 	type TransactionalProcessor: ProcessTransaction;
+
+	type HrmpNewChannelOpenRequestHandler: HandleHrmpNewChannelOpenRequest;
+	type HrmpChannelAcceptedHandler: HandleHrmpChannelAccepted;
+	type HrmpChannelClosingHandler: HandleHrmpChannelClosing;
```

Comment:
@JuaniRios please add some docs for all

Reply:
[094f6f4](https://github.com/paritytech/polkadot-sdk/pull/3696/commits/094f6f41857adee59d41c13e0fdcfb3db173d9f1)